packages/app/src/renderer/locales/default/en-US.json

```diff
@@ -664,6 +664,11 @@
   "module.settings.tetheringLabel": "Start tethering",
   "module.templates": "Templates",
   "module.templates.deleteButton": "Delete",
+  "module.templates.deleteModalBody": "Do you really want to delete <b>{num, plural, =-1 {all templates} one {# template} other {# templates}}?</b>",
```

Comment:
Just curious - is there any way/chance to test how the translation message would look like? At least for me, it looks a little bit complicated.

Reply:
I tried a quick check on this by test, but with no results. This translation is connected to Phrase, [here ](https://help.phrase.com/help/working-with-pluralization) you can find more about it. Generally, it is a kind of if statement.